README.md

```diff
@@ -27,9 +27,9 @@ Sandbox API Repository to describe, develop, document, and test the ApplicationE
 The repository has no (pre)releases yet, work in progress is within the main branch.
 <!-- Optional: an explicit listing of the latest (pre-)release with additional information, e.g. links to the API definitions -->
 <!-- In addition use/uncomment one or multiple the following alternative options when becoming applicable -->
-<!-- Pre-releases of this sub project are available in https://github.com/camaraproject/ApplicationEndpointDiscovery/releases -->
-<!-- The latest public release is available here: https://github.com/camaraproject/ApplicationEndpointDiscovery/releases/latest -->
-<!-- For changes see [CHANGELOG.md](https://github.com/camaraproject/ApplicationEndpointDiscovery/blob/main/CHANGELOG.md) -->
+Pre-releases of this sub project are available in https://github.com/camaraproject/ApplicationEndpointDiscovery/releases
+The latest public release is available here: https://github.com/camaraproject/ApplicationEndpointDiscovery/releases/latest
```

Comment:
There is not yet a public release ... so can consider to keep this line commented for now.